weaviate/collections/classes/config.py

```diff
@@ -278,6 +278,17 @@ class PQEncoderDistribution(str, Enum):
     NORMAL = "normal"
 
 
+class MultiVectorAggregation(str, Enum):
```

Comment:
don't we have a base string enum class? But looks like we don't use it anywhere else in this file